dynatrace-operator/README.md

```diff
@@ -41,6 +41,29 @@ Generate an API and a PaaS token in your Dynatrace environment.
 
 https://www.dynatrace.com/support/help/reference/dynatrace-concepts/why-do-i-need-an-environment-id/#create-user-generated-access-tokens
 
+### Installation
+
+#### Apply CustomResourceDefinition
+
+Download the latest version of the CRD attached to the latest release and apply it.
```

Comment:
We could add a link to the [release page](https://github.com/Dynatrace/dynatrace-operator/releases) of the operator, so the CRDs are easier to find.

Reply:
Thought about that but since we release the dynatrace-operator and the dynatrace-oneagent-operator within that repository pointing to the latest release does not work properly

Reply:
Good idea - added a link